pkgs/applications/misc/gkrellm/default.nix

```diff
@@ -1,8 +1,21 @@
 { lib, fetchurl, stdenv, gettext, pkg-config, glib, gtk2, libX11, libSM, libICE, which
```

Comment:
Since it is a gtk application it also like needs wrapGAppsHook to work correctly.

Reply:
Interesting!  Added, thanks.